File: RedTeam/Ruby/reverse_shell.rb
#!/usr/bin/ruby

# Author: ForzaxHX
# Ruby reverse shell

require 'shellwords'

def reverse_shell(command)
shellwords(command).reverse.join(' ')
end

if ARGV.length > 0
reverse_shell(ARGV[0])
else
  puts "Usage: reverse_shell <command>"
end

File: RedTeam/Perl/reverse_shell.pl
#!/usr/bin/perl

# Author: FrozaxHX
# Perl Reverse Shell

use strict;
use warnings;

my $in = shift || die "Usage: $0 input";
open my $fh, '<', $in or die "could not open '$in': $!";
while ( < $fh > ) {
  chomp;
  print $_;
}
close $fh;